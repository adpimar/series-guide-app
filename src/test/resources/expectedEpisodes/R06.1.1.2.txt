E#5878319#498968#1#4#Away#2016-12-16#Un doloroso dilema ofrece a OA una nueva perspectiva de los problemas del grupo. Homer decide averiguar el verdadero prop√≥sito de los experimentos de Hap.#T#null