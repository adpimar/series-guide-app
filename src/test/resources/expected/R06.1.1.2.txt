E#5878319#691955#1#4#Away#2016-12-16#Un doloroso dilema ofrece a OA una nueva perspectiva de los problemas del grupo. Homer decide averiguar el verdadero prop√≥sito de los experimentos de Hap.#F#null